src/engine/controls/cuecontrol.cpp

```diff
@@ -135,6 +135,10 @@ CueControl::CueControl(const QString& group,
     m_pCueIndicator = new ControlIndicator(ConfigKey(group, "cue_indicator"));
     m_pPlayIndicator = new ControlIndicator(ConfigKey(group, "play_indicator"));
 
+    m_pIsPlaying = new ControlPushButton(ConfigKey(group, "is_playing"));
+    m_pPlayProxi = new ControlProxy(ConfigKey(group, "play"));
+    m_pPlayProxi->connectValueChanged(this, &CueControl::setIsPlaying);
```

Comment:
this can be removed, because we already receive the play change signal by 
updateIndicatorsAndModifyPlay and updateIndicators. 

You may append a set() to m_pIsPlaying after every m_pPlayIndicator->setBlinkValue call and done.